ruby/rb-x/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           ruby 1.0
+
+ruby.branches       3.3 3.2 3.1
+ruby.setup          x 0.14.1 fetch
+github.setup        sferik x-ruby 0.14.1 v
+license             MIT
+categories-append   devel www
+platforms           any
+supported_archs     noarch
+maintainers         nomaintainer
+description         Ruby interface to the X (Twitter) API
+long_description    {*}${description}
+checksums           rmd160  61af8071c76aa881e4d38295abd0c2ca30cb24b2 \
+                    sha256  2a44417cdecf74852f19775b75897ee55298edee93dd285ec24b95e13b072bce \
+                    size    34707
+github.tarball_from archive
+
+if {${name} ne ${subport}} {
+    use_configure   no
+
+    build.cmd       ${ruby.gem}
+    build.target    build
+
+    build {
+        command_exec build
+    }
+
+    destroot.cmd        ${ruby.gem}
+    destroot.target     install
+    destroot.args       --local --force --install-dir ${destroot}${ruby.gemdir}
+    destroot.env-append rake=${ruby.rake}
+    destroot.post_args  x-${version}.gem
+
+    destroot {
+        command_exec destroot
```

Comment:
Same here, shouldn't this be removed and instead a `post-destroot` section with the code below added?